app/views/verify/address/index.html.slim

```diff
@@ -12,4 +12,4 @@ p
       = link_to t('idv.buttons.activate_by_mail'), verify_usps_path,
         class: 'btn btn-outline rounded-lg mb2 center'
 
-= render 'shared/cancel', link: verify_cancel_path
+= render 'shared/cancel', link: verify_cancel_path, decorated_session: decorated_session
```

Comment:
No need to pass in the decorated_session here since it's already available to the view automatically by being defined as a `helper_method` in ApplicationController.